### DISCUSSION TITLE
Disable list

### DISCUSSION THREAD

**Discussion by aliakbarmotallebi** [community/community#8699] [Other Feature Feedback, Questions, & Ideas]
The choice must be made not to show the stars to others
It is not interesting to be active only without the owner's permission

**Comment by brianlovin** [comment]
Hey @aliakbarmotallebi could you share more about this? I couldn't quite understand what you meant, or what the proposed change is. Thanks!

**Reply by gitatmax** [reply]
I'm not 💯 but I think they're referring to #10742

**Reply by aliakbarmotallebi** [reply]
hi my friend
Disable the ability to display starred reservoirs

**Reply by aliakbarmotallebi** [reply]
for users other
